.github/workflows/build-executable.yml

```
@@ -0,0 +1,147 @@
+            name: lace-linux-arm64
+          - os: macos-latest
+            target: bun-darwin-arm64
+            name: lace-macos-arm64
+          - os: macos-13
+            target: bun-darwin-x64
+            name: lace-macos-x64
+    
+    runs-on: ${{ matrix.os }}
+    
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)